### DISCUSSION TITLE
Error: The `initialStatus` argument for `Prerender` must be a natural number.

### DISCUSSION THREAD

**Discussion by McKayMower** [vercel/community#2570] [Help]
### Summary

I am getting the error in the title when building on vercel. Building locally there are no errors, and running lint also produces no errors. 

### Additional information

Dependencies:
"dependencies": {
    "@next-auth/prisma-adapter": "^1.0.6",
    "@prisma/client": "^4.14.1",
    "@types/node": "20.2.3",
    "@types/react": "18.2.6",
    "@types/react-dom": "18.2.4",
    "autoprefixer": "10.4.14",
    "axios": "^1.4.0",
    "class-variance-authority": "^0.6.0",
    "clsx": "^1.2.1",
    "eslint": "8.41.0",
    "eslint-config-next": "13.4.3",
    "mongodb": "^5.5.0",
    "next": "13.4.3",
    "next-auth": "^4.22.1",
    "postcss": "8.4.23",
    "prisma": "^4.14.1",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "react-icons": "^4.8.0",
    "react-spinners": "^0.13.8",
    "tailwind-merge": "^1.12.0",
    "tailwindcss": "3.3.2",
    "typescript": "5.0.4"
  }

Scripts: 
"scripts": {
    "dev": "next dev",
    "build": "next build",
    "postinstall": "prisma generate",
    "start": "next start",
    "lint": "next lint"
  },

Error logs from vercel deployment status:
<html>
<body>

Running build in Washington, D.C., USA (East) – iad1
--
00:20:21.049 | Cloning github.com/McKayMower/* (Branch: Dev, Commit: *)
00:20:21.057 | Skipping build cache, deployment was triggered without cache.
00:20:21.442 | Cloning completed: 392.871ms
00:20:21.620 | Running "vercel build"
00:20:22.281 | Vercel CLI 29.3.6
00:20:22.589 | Installing dependencies...
00:20:38.735 |  
00:20:38.736 | > *@0.1.0 postinstall
00:20:38.736 | > prisma generate
00:20:38.736 |  
00:20:39.560 | Environment variables loaded from .env
00:20:39.564 | Prisma schema loaded from prisma/schema.prisma
00:20:40.385 |  
00:20:40.385 | ✔ Generated Prisma Client (4.14.1 \| library) to ./node_modules/@prisma/client in 168ms
00:20:40.385 | You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
00:20:40.386 | ```
00:20:40.386 | import { PrismaClient } from '@prisma/client'
00:20:40.386 | const prisma = new PrismaClient()
00:20:40.386 | ```
00:20:40.444 |  
00:20:40.444 | added 396 packages in 17s
00:20:40.444 |  
00:20:40.444 | 142 packages are looking for funding
00:20:40.444 | run `npm fund` for details
00:20:40.471 | Detected Next.js version: 13.4.3
00:20:40.478 | Detected `package-lock.json` generated by npm 7+...
00:20:40.479 | Running "npm run build"
00:20:40.879 |  
00:20:40.880 | > *l@0.1.0 build
00:20:40.880 | > next build
00:20:40.880 |  
00:20:41.250 | - info Loaded env from /vercel/path0/.env
00:20:41.426 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
00:20:41.426 | This information is used to shape Next.js' roadmap and prioritize features.
00:20:41.426 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
00:20:41.426 | https://nextjs.org/telemetry
00:20:41.426 |  
00:20:41.551 | - info Creating an optimized production build...
00:21:01.401 | - info Compiled successfully
00:21:01.405 | - info Linting and checking validity of types...
00:21:06.084 | - info Collecting page data...
00:21:22.626 | - info Generating static pages (0/8)
00:21:22.742 | - info Generating static pages (2/8)
00:21:52.759 | - info Generating static pages (4/8)
00:21:52.990 | - info Generating static pages (6/8)
00:21:53.063 | - info Generating static pages (8/8)
00:21:53.230 | - info Finalizing page optimization...
00:21:53.234 |  
00:21:53.260 | Route (app)                                Size     First Load JS
00:21:53.260 | ┌ ○ /                                      873 B          95.5 kB
00:21:53.260 | ├ λ /api/auth/[...nextauth]                0 B                0 B
00:21:53.260 | ├ ○ /api/*                          0 B                0 B
00:21:53.261 | ├ ○ /auth/login                            8.65 kB         104 kB
00:21:53.261 | ├ ○ /favicon.ico                           0 B                0 B
00:21:53.261 | ├ ○ /*                            20.7 kB         105 kB
00:21:53.261 | └ ○ /tempsensitiveroute                    136 B          77.3 kB
00:21:53.261 | + First Load JS shared by all              77.2 kB
00:21:53.261 | ├ chunks/139-bf56990e3ee9b325.js         24.7 kB
00:21:53.261 | ├ chunks/2443530c-f1d3428dc335824e.js    50.5 kB
00:21:53.261 | ├ chunks/main-app-c0fde4ac3a0b3bab.js    215 B
00:21:53.261 | └ chunks/webpack-f9a4eed9b874fd0b.js     1.74 kB
00:21:53.261 |  
00:21:53.262 | Route (pages)                              Size     First Load JS
00:21:53.263 | ─ ○ /404                                   178 B            84 kB
00:21:53.263 | + First Load JS shared by all              83.9 kB
00:21:53.263 | ├ chunks/main-0467611a12ccfd1b.js        81.9 kB
00:21:53.263 | ├ chunks/pages/_app-c544d6df833bfd4a.js  192 B
00:21:53.263 | └ chunks/webpack-f9a4eed9b874fd0b.js     1.74 kB
00:21:53.263 |  
00:21:53.263 | ƒ Middleware                               23.7 kB
00:21:53.263 |  
00:21:53.264 | λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
00:21:53.264 | ○  (Static)  automatically rendered as static HTML (uses no initial props)
00:21:53.264 |  
00:21:57.808 | Traced Next.js server files in: 3.786s
00:22:07.138 | Created all serverless functions in: 9.327s
00:22:07.224 | Error: The `initialStatus` argument for `Prerender` must be a natural number.
00:22:07.919 | Deployment completed
00:22:07.474 | : The `initialStatus` argument for `Prerender` must be a natural number.

</body>
</html>

**Comment by amyegan** [comment]
Sorry you're running into this build error, @McKayMower. Are you using `npm run build` or [`vercel build`](https://vercel.com/docs/cli/build#) for local builds?

**Reply by McKayMower** [reply]
I am using npm run build

**Reply by amyegan** [reply]
Do you get the error if you build with the latest version of [Vercel CLI](https://www.npmjs.com/package/vercel)? You can use `vercel build --debug` to output extra information

**Reply by smaeda-ks** [reply]
should 301 to https://github.com/vercel/next.js/discussions/50497
